utils/check_tags.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
I executed that script on every md files and it seams to have some glitches.  For instance, it says that "medical" and "reinforcement" are no valid tags which they are.

Reply:
Thanks for testing ! It does not seem to happen on my end. Does this happen on every file where these tags exist, or it just points them out in some files?

The style is not consistent across our files and may be the script needs to be more robust to these variations (commas, double quotes, etc.)

Reply:
For some bizzare reasons, every tags are ok in 2017-05-08-resunet.md  but not in 2019-01-17-MixedSupervisionBrainTumorSegmentation.md.  In the latter, the tag "medical" is flaged as missing.  What is weird is that the same flag is in 2017-05-08-resunet.md.
 
in the md files:
Erroneous case:
tags: deep-learning multi-task-learning segmentation medical

correct case 
tags:   deep-learning CNN segmentation medical

The only difference I see are the spaces between "tags:" and "deep-learning.

Reply:
Thanks ! That will help me debug the script ! Will keep you posted.

Reply:
As for the tags, I've done a few tests. Immediately after the `files=("$@")` statement I added `IFS=$' ' read -r -a files <<< $(find article/_posts -name "*.md")` to the script to give it the complete list of our review files and executed the script (with a dummy first argument). I can confirm the behavior you observed. However, I found out that it is due to some "invisible" characters that at times are present in some of the tag lines. As far as I've found, they are the Windows CR characters: if, for example, you remove ` medical` from the tags list of one of the files spotted by the script, the immediate anterior tag will be marked as missing.

At times I've started my reviews on my Windows laptop, so that may explain things. Looking for CR in review files points to my reviews only:
```
$ find article/_posts -name "*.md" | grep -l -P '\x0d' article/_posts/*.md
```

So I'm sorry.

I've done a sweep to change `CRLF`s to `LF`s: #376

I've executed the `check_tags.sh` script once the line-endings changed, and the above behavior has been fixed :heavy_check_mark:. I'd dare to say that the rest of the files the script identifies are real cases of missing/misspelled tags.

I've added a `.gitattributes` file to avoid these situations: #377

I've checked on one of the files that has `CRLF` endings, and `.gitattributes` is doing its job.

Apart from that, the missing tag identification is case-sensitive, which I guess is correct :thumbsup:.